packages/amplify-category-storage/provider-utils/awscloudformation/service-walkthroughs/s3-walkthrough.js

```diff
@@ -357,10 +357,11 @@ async function configure(context, defaultValuesFilename, serviceMetadata, resour
       switch (triggerOperationAnswer.triggerOperation) {
         case 'Update the Trigger': {
           try {
+            await removeTrigger(context, parameters.resourceName, parameters.triggerFunction);
             answers.triggerFunction = await addTrigger(
               context,
               parameters.resourceName,
-              parameters.triggerFunction,
+              undefined,
```

Comment:
There are 2 references to `addTrigger` function and after this update `undefined` is sent from both the places. Can you update the `addTrigger` by removing this parameter